server/sonar-web/src/main/webapp/WEB-INF/app/models/dashboard.rb

```diff
@@ -55,16 +54,8 @@ def shared?
     read_attribute(:shared) || false
   end
 
-  def global=(global)
-    write_attribute(:is_global, global)
-  end
-
-  def global
-    read_attribute(:is_global)
-  end
-
   def global?
```

Comment:
to be fully backward-compatible, you can also keep 
```
def global
  true
end
```

No need for keeping `global=`